package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "hostile",
   "description": "Simple /etc/hosts manipulation",
-  "version": "1.0.3",
+  "version": "1.1.3",
```

Comment:
Please don't change the version in PRs. That's the job of the publisher to do at publish time.